### ISSUE TITLE
ROS release?

### ISSUE THREAD

**Issue by simonschmeisser** [tesseract-robotics/trajopt#300]
I'm currently spending some time on integrating trajopt as a MoveIt planner and was wondering if there are plans for a release to the ros noetic build farm? I have seen tesseract has been released already.

**Comment by simonschmeisser** [comment]
https://github.com/ros-planning/moveit/pull/3235 is my WIP branch for adding collision terms

**Comment by simonschmeisser** [comment]
Yes that's another attractive approach, I'll try to implement that as well to see if I can reproduce your collision checking performance plot (or do you have the source somewhere?)

(tesseract_planning has not been released either, right?)

**Comment by Levi-Armstrong** [comment]
It is not, mostly because some of the dependencies are not release like [osqp](http://repositories.ros.org/status_page/ros_noetic_default.html?q=osqp).  I reached out to @esteve to see about being added as maintainer to fix the issue. I think after this is release it should be possilble to release tesseract_planning and tesseract_ros.

**Comment by simonschmeisser** [comment]
Hi Levi,
if I understood things correctly osqp has been released in the mean time?

**Comment by Levi-Armstrong** [comment]
I believe so.
